internal/gmtcli/config.go

```diff
@@ -36,8 +36,51 @@ func printConfig(r gomesh.Radio) error {
 		return err
 	}
 
+	for _, config := range configSettings {
```

Comment:
I understand the thought behind relying on the protobufs to handle displaying information. The info is already there and we don't have to manually update things if there are changes! That was how I originally built the library, but some of the protobufs are really hard to digest so I moved to the more manual listing below. 

I'd prefer to keep the human readable format, if you can find a way to do that and rely less on manual listing of fields I'm all for it!